src/Illuminate/Mail/Message.php

```diff
@@ -68,11 +68,16 @@ public function returnPath($address)
 	 * Add a recipient to the message.
 	 *
 	 * @param  string|array  $address
-	 * @param  string  $name
+	 * @param  string|null   $name
+	 * @param  boolean       $override   Will force ignoring the previous recipients
```

Comment:
Also, could you move this argument description to be the "Long description" please.